Feisty/MockResources/gameList.json

```diff
@@ -3,113 +3,43 @@
     "apps": [
```

Comment:
This file was completely wrong because it did not mirror the actual response properly. All the removed parts have to be outside of this call as the actual service call's response follows a different structure.